### PR TITLE
Release notes for Feb 26th

### PR DESCRIPTION
**Pull request by justindavies**

### CODE REVIEW

**Review by miwithro — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,40 @@
...
+  * The Max Surge default value will change on newly created nodepools from 1 to 10%.
+* AKS began pod security policy deprecation on 2022-11-01 API. The [pod security policy](https://learn.microsoft.com/azure/aks/use-pod-security-policies) will be removed completely on 2023-06-01 API with AKS 1.25 version or higher. You can migrate pod security policy to [pod security admission controller](https://learn.microsoft.com/azure/aks/use-psa) before the deprecation deadline.
+
+### Release notes
+
+* Preview Features
+  * Support for (Pod Sandboxing)[https://learn.microsoft.com/en-gb/azure/aks/use-pod-sandboxing] workloads
+  * Enable windows metrics collection from the Azure Monitor Metrics
+* Bug Fix
+  * In 2023-01-01 Azure API, a hot fix is released and currently rolling out to fix this (bug)[https://github.com/Azure/AKS/issues/3481] and returns 400 error on PUT requests to "Base" or "Standard" parameters, allowing customers to still use "Basic" parameter in ManagedClusterSKUName with "Free" or "Paid" parameters in ManagedClusterSKUTier.
```
Comment (wangyira) [inline CHANGELOG.md:25]: Could you please take out "and currently rolling out", the release is complete, thanks

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,40 @@
...
+  * Fix cluster autoscaler scheduler bug which is causing CA to crash. 
+  * Update node label with Security Patch versions from VHD 
+* Behavior Changes
+  * Removed 5 minute back off when attemptng to delete a node pool with an existing operation taking place
+* Component Updates
+  * Azure Blob CSI driver updated to version v1.19.1
+  * Update Prometheus Add-on to [02-22-2023](https://github.com/Azure/prometheus-collector/blob/main/RELEASENOTES.md#release-02-22-2023)
+  * AKS Windows 2019 image has been updated to [17763.4010.230223](vhd-notes/AKSWindows/2019/17763.4010.230223.txt).
+  * AKS Windows 2022 image has been updated to [20348.1547.230223](vhd-notes/AKSWindows/2022/20348.1547.230223.txt).
+
```
Comment (wangyira) [inline CHANGELOG.md:37]: did Ubuntu 18.04 and 22.04 and Mariner get updated?

**Review by wangyira — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (justindavies) [reply CHANGELOG.md:37]: No

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,40 @@
...
+  * Core DNS changes - Update to 1.10 and will set lamemark to 5s & ttl to 30 to follow community defaults
+  * HostProcess Containers will be GA
+  * Some AKS labels will be deprecated. Update your AKS labels to the recommended substitutions. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Starting with Kubernetes 1.27:
+  * The Max Surge default value will change on newly created nodepools from 1 to 10%.
+* AKS began pod security policy deprecation on 2022-11-01 API. The [pod security policy](https://learn.microsoft.com/azure/aks/use-pod-security-policies) will be removed completely on 2023-06-01 API with AKS 1.25 version or higher. You can migrate pod security policy to [pod security admission controller](https://learn.microsoft.com/azure/aks/use-psa) before the deprecation deadline.
+
+### Release notes
+
+* Preview Features
```
Comment (kaarthis) [inline CHANGELOG.md:21]: Node OS auto-upgrade channel provides an exclusive way for automatically getting OS security patches promptly.  This new channel works in tandem with existing Autoupgrade channel which does K8s upgrades. https://learn.microsoft.com/en-us/azure/aks/auto-upgrade-node-image . 
Pls include in preview section.

**Review by kaarthis — APPROVED** [review]

**Review by wangyira — APPROVED** [review]

**Pull request merged**